src/types/modules/react-native-web-linear-gradient.d.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable @typescript-eslint/no-empty-interface */
```

Comment:
@kubabutkiewicz Why do we need to disable this?

Reply:
@VickyStash I added that because we need to use interfaces here since the class is exported from that module and modules can extends only from interfaces (also in other custom ts modules it was done in that way 😄 )

Reply:
@kubabutkiewicz Yeah, but you don't have the empty interface, so I guess you can remove this 
`/* eslint-disable @typescript-eslint/no-empty-interface */`

Reply:
Lol, yeah I can't read , thanks ! 😅